src/components/builder/values/values-config.stories.tsx

```diff
@@ -300,3 +303,57 @@ export const SelectVariable: SelectStory = {
     },
   },
 };
+
+export const SelectReferentielijsten: SelectStory = {
+  ...Select,
+
+  decorators: [withFormik, BuilderContextDecorator],
```

Comment:
I think we can move this up into the meta instead, I'm not sure why I ever decided this should go in a base story thing :thinking: 

They all need access to formik afaik.

Reply:
Adding `decorators` to the default story seems to cause errors: https://github.com/open-formulieren/formio-builder/actions/runs/13134517347/job/36646878514?pr=204#step:7:2152